transports/p2p/addrmgr/addrmanager.go

```diff
@@ -211,6 +227,42 @@ func (a *AddrManager) updateAddress(netAddr, srcAddr *wire.NetAddress) {
 	a.Log.Trace().Msgf("Added new address %s for a total of %d addresses", addr, a.nTried+a.nNew)
 }
 
+// BanAddress creates a ban for defaultBanTime for a peer that we cannot connect to.
+func (a *AddrManager) BanAddress(addr string) {
+	a.addrBanned[addr] = time.Now().Add(defaultBanTime)
+
+	// remove from addrTried buckets
```

Comment:
I know that I repeat myself a lot about this, but instead of those two comments I would create two separated private funcs to handle it - it will be more descriptive for me :D

Reply:
**MINOR**

Reply:
You're right, done :)